src/layer/layer.js

```diff
@@ -27,15 +27,21 @@ class LayerComponent extends React.Component<
   state = {container: null};
 
   componentDidMount() {
-    const {onMount, mountNode} = this.props;
+    const {onMount, mountNode, host: layersManagerHost} = this.props;
     if (mountNode) {
       onMount && onMount();
       return;
     }
+
     // There was no LayersManager added if this.props.host === undefined.
     // Use document.body is the case no LayersManager is used.
-    const host =
-      this.props.host !== undefined ? this.props.host : document.body;
+    const hasLayersManager = layersManagerHost !== undefined;
+    if (!hasLayersManager) {
```

Comment:
Can you please wrap this in a `__DEV__` conditional? There should be other instances if you search the codebase. Doing so will exclude this check from production builds - reducing bundle size and some excess operations at runtime

Reply:
oops good catch, thanks!